app.py

```diff
@@ -1,21 +1,20 @@
 # Import libraries
 import numpy as np
 import pandas as pd
-from sklearn import grid_search
-from sklearn.cross_validation import train_test_split
 
 from lib.data.wrangler import readData
 from lib.helpers import runTests, writeToCsv
 
-X_test, \
-    X_train, \
+X_train, \
+    X_test, \
     y_test, \
     y_train = readData('./tmp/testTrainData.npz')
 
 classifiers, \
     train_times, \
     pred_times, \
     f1_trains, \
-    f1_tests = runTests(X_test, X_train, y_test, y_train)
+    f1_tests, \
+    gs_times = runTests(X_test, X_train, y_test, y_train)
 
-writeToCsv(classifiers, train_times, pred_times, f1_trains, f1_tests)
+writeToCsv(classifiers, train_times, pred_times, f1_trains, f1_tests, gs_times)
```

Comment:
We need a common style guide at least for naming things.
Either `snake_case` or `camelCase` for function_names.
variables as `smallnocaps`
Classes as CapitalCase.